provisioner/aws.go

```diff
@@ -771,6 +771,7 @@ func tagsToMap(tags []*ec2.Tag) map[string]string {
 type EKSClusterInfo struct {
 	Endpoint             string
 	CertificateAuthority string
+	OIDCIssuerURL        string
 }
 
 func (a *awsAdapter) GetEKSClusterCA(cluster *api.Cluster) (*EKSClusterInfo, error) {
```

Comment:
We should rename this method to something like `GetEKSClusterDetails`. The method return more than the CA Data. I suggest the suffix "Details" because that's where this data shows is in the Overview of the cluster in the AWS Console.